frontend/src/components/funding/FundingAllocationInfoTable.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <v-container fluid class="pa-0">
+    <div v-if="$vuetify.display.mdAndUp" class="table">
```

Comment:
You could have a single div with class="table" and use templates for your if/else. It would save the class duplication but add more tags. Either way is fine.

Reply:
it's updated.